editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1705,7 +1698,8 @@ SpriteFramesEditor::SpriteFramesEditor() {
 	sub_vb->add_child(animations);
 	animations->set_v_size_flags(SIZE_EXPAND_FILL);
 	animations->set_hide_root(true);
-	animations->connect("cell_selected", callable_mp(this, &SpriteFramesEditor::_animation_selected));
+	// HACK: The cell_selected signal is emitted before the FPS spinbox loses focus and applies the change.
+	animations->connect("cell_selected", callable_mp(this, &SpriteFramesEditor::_animation_selected), CONNECT_DEFERRED);
```

Comment:
Regarding the hack itself (and the same applies to https://github.com/godotengine/godot/pull/79872 really): we could probably avoid this if we store which item was selected when focus was gained, and on focus loss we update that item, and not the current item.

What do you think about that?

Reply:
This will work, but I think it's about the same hack as this one, but also requires adding an unnecessary variable with duplicate content, a callback, and a connection to the `LineEdit` signal. I would prefer to keep it as is until #79695 is fixed or if we find any problems with this solution.